devtools/azure/pipelines.yml

```diff
@@ -0,0 +1,20 @@
+jobs:
+  - job: UnitTest
```

Comment:
We can only run UnitTest if

- It is triggered by PR
- Or it is triggered for master branch

Here is the condition used in travis, FYI

```
if: 'tag IS NOT present AND (type == pull_request OR branch = master OR branch =~ /^test/)'
```

Reply:
The logic has been fixed except for the tagging part, I couldn't find one variable containing git tag in Azure Pipeline conditions.